suite-common/wallet-core/src/discovery/passphraseUtils.ts

```diff
@@ -17,6 +17,11 @@ export const determinePassphraseFlowState = (
         (!modalState.modalContextDevice || modalState.context === modalState.modalContextDevice) &&
         modalState.windowType === UI.REQUEST_PASSPHRASE
     );
+
+    const isAddingHiddenWalletWithRespectToSettings = Boolean(
+        discovery.isAddingHiddenWalletWithRespectToSettings,
+    );
```

Comment:
I'd prefer explicit. It's really a nit, but the principle I follow is: handling falsy as false makes sense, but future refactorings can make unwanted truthy values silently handled as true (while with === TS will give error)
```suggestion
    const isAddingHiddenWalletWithRespectToSettings =
        discovery.isAddingHiddenWalletWithRespectToSettings === true;
```